pkg/services/featuremgmt/registry.go

```diff
@@ -1219,6 +1219,12 @@ var (
 			FrontendOnly:   false,
 			AllowSelfServe: false,
 		},
+		{
+			Name:        "pluginProxyPreserveTrailingSlash",
+			Description: "Preserve plugin proxy trailing slash.",
+			Stage:       FeatureStageExperimental,
```

Comment:
Instead of an experimental feature, could we consider this a bug fix?

Reply:
You can add `Expression: "true"` to enable it by default. In case of error, it's easier to change a flag than rolling back or pinning instances so it's safer this way.

Reply:
It's enabled by default now